src/sprites.py

```diff
@@ -224,12 +277,13 @@ def input(self):
                 if self.current_tool in {'hoe', 'axe'}:
                     self.sounds['swing'].play()
 
-            # seed switch 
+            # seed switch
             if recent_keys[pygame.K_e]:
-                self.seed_index = (self.seed_index + 1) % len(self.available_seeds)
+                self.seed_index = (self.seed_index +
+                                   1) % len(self.available_seeds)
```

Comment:
This is also a bit strange, but I guess the parentheses need to be set like this so that Python recognizes the newline?

Reply:
Agreed; I'll fix that.